### ISSUE TITLE
boundary sounds should reflect the enabled range, not total

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#248]
While using both hands when the ratio is locked, of the 4 "border" cases in that interaction (antecedent/up-down and consequent/up-down), only the consequent-up triggered border alerts (when locked at 1/2). All others are not at the very end, so they don't get border alerts. @emily-phet please let me know if this is a problem that should be solved.

**Comment by emily-phet** [comment]
The bottom position is 0 - if the lock ratio is on, then the current state has the Challenge Ratio and 0 should never be reached (though the hands should to a position just above 0. So the hands should never reach the bottom position. 

So, if I'm understanding correctly the cases where the border alert is triggered, I think this sounds correct to me. 

I would guess this will come up in QA as a potential bug, so if you have a way of recording known quirkiness to the QA team, this should probably go on the list.

**Comment by zepumph** [comment]
>So, if I'm understanding correctly the cases where the border alert is triggered, I think this sounds correct to me.


While you are right, from an interaction standpoint, I still feel like it may warrant further discussion. If I were blind, I may want feedback that my keypress didn't do anything, even though in other modes I could go further.

Right now, it is possible to continue to press down and hear nothing (because repeat description context responses won't be said). Potentially after work in https://github.com/phetsims/ratio-and-proportion/issues/220#issuecomment-733245324, this may turn into repeating  something like "both hands near bottom, at challenge ratio". I still feel like more could be done to demonstrate that the keypress yielded no change in the sim. 

Thoughts?

**Comment by zepumph** [comment]
During today's design meeting, @terracoda mentioned that for description, there are edge descriptions given for these cases when the ratio is locked. See https://github.com/phetsims/ratio-and-proportion/issues/247. 

We feel like this should stay as it is for pedagogical reasons. @emily-phet will reach out to our research partner to see if anything should change.
